ecs.php

```diff
@@ -524,5 +525,7 @@
                 'PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ScopeClosingBraceSniff.ContentBefore' => null,
             ]
         );
+
+        $parameters->set(Option::PARALLEL, true);
```

Comment:
@MortalFlesh is this necessary? symplify/easy-coding-standard:^10 seems to enable this by default as well :)